kore/app/share/GlobalMain.hs

```diff
@@ -295,17 +298,28 @@ commandLineParse
     :: Parser a                -- ^ local options parser
     -> InfoMod (MainOptions a) -- ^ local parser info modifiers
     -> IO (MainOptions a)
-commandLineParse localCommandLineParser modifiers =
-    execParser
-    $ info
-        ( MainOptions
-            <$> globalCommandLineParser
-            <*> (   Just <$> localCommandLineParser
-                <|> pure Nothing
+commandLineParse localCommandLineParser modifiers = do
+    args' <- Env.getArgs
+    env <- Env.lookupEnv "KORE_EXEC_OPTS"
+    let
+        args = case env of
+            Nothing -> args'
+            Just opts -> args' <> words opts
```

Comment:
I think the environment options should come first, so that the command-line argument override.

How will users specify command-line arguments with spaces using `KORE_EXEC_OPTS`?

Reply:
We will forget I asked these questions.